checker/src/org/checkerframework/checker/genericeffects/GenericEffectExtension.java

```
@@ -0,0 +1,149 @@
+    public Class<? extends Annotation> checkContinue(ContinueTree node) { return null; }
+    public Class<? extends Annotation> checkDoWhileLoop(DoWhileLoopTree node) { return null; }
+    public Class<? extends Annotation> checkEnhancedForLoop(EnhancedForLoopTree node) { return null; }
+    public Class<? extends Annotation> checkForLoop(ForLoopTree node) { return null; }
+    public Class<? extends Annotation> checkIf(IfTree node) { return null; }
+    public Class<? extends Annotation> checkInstanceOf(InstanceOfTree node) { return null; }
+    public Class<? extends Annotation> checkIntersectionType(IntersectionTypeTree node) { return null; }
+    public Class<? extends Annotation> checkLabeledStatement(LabeledStatementTree node) { return null; }
+    public Class<? extends Annotation> checkLiteral(LiteralTree node) { return null; }
+    public Class<? extends Annotation> checkNewArray(NewArrayTree node) { return null; }
+    public Class<? extends Annotation> checkPrimitiveType(PrimitiveTypeTree node) { return null; }
+    public Class<? extends Annotation> checkReturn(ReturnTree node) { return null; }
+    public Class<? extends Annotation> checkSwitch(SwitchTree node) { return null; }
+    public Class<? extends Annotation> checkSynchronized(SynchronizedTree node) { return null; }
+    public Class<? extends Annotation> checkThrow(ThrowTree node) { return null; }
+    public Class<? extends Annotation> checkTry(TryTree node) { return null; }
+    public Class<? extends Annotation> checkTypeCast(TypeCastTree node) { return null; }
+    public Class<? extends Annotation> checkUnary(UnaryTree ndoe) { return null; }
+    public Class<? extends Annotation> checkUnionType(UnionTypeTree node) { return null; }
+    public Class<? extends Annotation> checkWhileLoop(WhileLoopTree node) { return null; }
+    public Class<? extends Annotation> checkWildcard(WildcardTree node) { return null; }
```

Rather than returning ```null```, these methods should throw ```java.lang.UnsupportedOperationException``` (https://docs.oracle.com/javase/7/docs/api/java/lang/UnsupportedOperationException.html).  They shouldn't be called at all if the corresponding ```doesXCheck()``` returns false.